app/services/paypal_service/data_types/ipn.rb

```diff
@@ -248,6 +248,8 @@ def to_authorization_created(params)
           },
           params)
 
+        p[:order_id] = p[:order_id].strip.empty? ? nil : p[:order_id]
```

Comment:
Empty string maps to None, so you could also use:

`Maybe(p)[:order_id].strip.or_else(nil)`